db/migrations/20220414100600.json

```diff
@@ -0,0 +1,4 @@
+{
+  "up": "UPDATE people SET approved = true",
+  "down": "UPDATE people SET approved = NULL"
+}
```

Comment:
This has been moved to @klaudsol/commons
https://github.com/klaudsol/klaudsol-commons/blob/main/dist/db/migrations/20220414100600.json